aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -1124,9 +1142,12 @@ Tensor reduce_sparse_csr_dim1_cpu_template(const Tensor& sparse, ReductionOp rop
     new_col_indices.resize_(nnz);
     new_col_indices.fill_(index_t(0));
     new_values.resize_(nnz);
+    if (!new_values_acc.is_same(new_values)) {
```

Comment:
Thank you for addressing the comments. Is there something you can do to abstract away this line as well? It's repeat 5 times.